webapi/src/main/scala/org/knora/webapi/util/search/v2/SearchParserV2.scala

```diff
@@ -192,7 +195,7 @@ object SearchParserV2 {
           * @return a [[Entity]].
           */
         private def makeEntity(objVar: algebra.Var): Entity = {
```

Comment:
I think this method should add the entity to `positiveEntities` if it's an `IriRef` or a `QueryVariable`, but not if it's an `XsdLiteral`.

Reply:
right, an `XsdLiteral` makes no sense because we would not check for it later